src/monster.h

```diff
@@ -303,8 +304,11 @@ class monster : public Creature, public JsonSerializer, public JsonDeserializer
         std::string unique_name; // If we're unique
         bool hallucination;
 
-        bool setpos(const int x, const int y, const bool level_change = false);
-        bool setpos(const point &p, const bool level_change = false);
+        // level_change == true means "monster isn't spawned yet, don't update position in tracker"
```

Comment:
The `level_change` argument will be going away once z-level update is finished, that's why I didn't clean it up yet.
It is only used in `monster_stair_update()` code.
I didn't change it to `spawn`, because later on `spawn` might (probably should) do some extra checks, like submerging the monster in water.